packages/sui-js/src/browser/index.js

```diff
@@ -0,0 +1,11 @@
+import bowser from 'bowser'
+
+const browser = (userAgent) => {
```

Comment:
I think that is API is too strict and hard to involved.
I would prefer put this behind an Object.
In this first version:
```js
export default {
  stats: au
}
```
And use it in this way:
`import {stats} from '@s-ui/js/browser'`
And later be apply put more helper with touch the public API

```js
export default {
  stats: au,
  cookies: [What_ever_cookie_parser_go_here]
}
```
And use it:
`import {stats, cookies} from '@s-ui/js/browser'`

Reply:
Done @carlosvillu . Check it out!